packages/docs/pages/reference/elements/p.mdx

```diff
@@ -7,7 +7,8 @@ import {
 
 # `<p>`
 
-Short description of what the `p` element does.
+A paragraph, which can contain text and a wide variety of inline tags, and
```

Comment:
Add something about the `p` being the "workhorse" element, or at least really common